Add more user interfaces to docs (#744)
nerves-project/nerves: docs/User Interfaces.md

mix firmware.burn
```

## Scenic

> Scenic is an graphical framework written directly on the Elixir/Erlang/OTP
> stack that allows you to do 2D rendering on screens. It is designed with

Reviewer: ```suggestion
Scenic is 2D UI framework written in Elixir that's designed with
```

embedded systems in mind and works well with Nerves on screens like the
[Raspberry Pi Touch Display](https://www.raspberrypi.com/products/raspberry-pi-touch-display/)
or HDMI connected screens.

Helpful links:
* [Scenic: Getting Start with Nerves doc](https://hexdocs.pm/scenic/getting_started_nerves.html)
* [Scenic Forum](https://elixirforum.com/c/elixir-framework-forums/scenic-forum/107)
* [ElixirConf 2018 - Introducing Scenic A Functional UI Framework - Boyd Multerer](https://www.youtube.com/watch?v=1QNxLNMq3Uw)
* [Scenic Now and Looking Ahead - Boyd Multerer | ElixirConfEU Virtual 20](https://www.youtube.com/watch?v=tej-SyhZrqk)

## eInk displays

Some initial work has been done to support eInk displays like the Pimoroni Inky
[pHAT](https://shop.pimoroni.com/products/inky-phat) and
[wHAT](https://shop.pimoroni.com/products/inky-what) models. Look at the
[`:inky` repo](https://github.com/pappersverk/inky) for more info.

## OLED

Basic work has been done to support small OLED screens with the SSD1306 chip which
are usually smaller screens a few inches wide. More info in the [`:oled` docs](https://hexdocs.pm/oled)

## Web Kiosks

> There have been many attempts over the years to add support for kiosk browsers
> on Nerves devices, but each has gone unmaintained. If this is of interest to

Reviewer: ```suggestion
Several companies have integrated web browsers with Nerves for kiosk applications.
Unfortunately, public repositories are currently unmaintained. If this is of interest to
```
